pepper-apis/src/main/java/org/broadinstitute/ddp/export/DataExporter.java

```diff
@@ -1075,14 +1105,35 @@ public int exportDataSetAsCsv(StudyDto studyDto, List<ActivityExtract> activitie
         List<String> headers = new LinkedList<>();
         headers.addAll(participantMetaFmt.headers());
 
+        Map<String, Integer> normalizedMaxInstanceCounts = new HashMap<>();
```

Comment:
Could you rename var to hint at what the key is for map? I think it would be instanceGuid, but I am making educated guess.

Reply:
This variable is well-contained inside this one method and usage has been mapping "activity tag" to an integer, as the code indicates. I can rename it to be extra explicit.

Reply:
Done

Reply:
Mentioning it because I was trying to figure it out and found myself navigating code to do so. Renaming would save the trouble next time.